src/module/actor/lancer-actor.ts

```diff
@@ -733,6 +735,11 @@ export class LancerActor extends Actor {
     return await flow.begin();
   }
 
+  async beginStructureFlow(_title?: string): Promise<boolean> {
+    const flow = new StructureFlow(this, undefined);
```

Comment:
Should we still override the title in this case since different structure rolls lead to drastically different effects and the titles are pulled from the roll table?

Reply:
That's fair. It does make sense for structure rolls to always pull the message title from the table (and if/when someone overrides the table to customize, it would just pull from the new table). In that case, I think we can just remove the `_title` arg.